Objects/longobject.c

```diff
@@ -2820,6 +2820,59 @@ that triggers it(!).  Instead the code was tested by artificially allocating
 just 1 digit at the start, so that the copying code was exercised for every
 digit beyond the first.
 ***/
+
+// Tables are computed by Tools/scripts/long_conv_tables.py
+#if PYLONG_BITS_IN_DIGIT == 15
```

Comment:
I'd consider putting the generated code in a `.c.h` file and `#include` it here, but up to you.

Reply:
IMHO, it's not quite enough lines to justify a separate file (adds about 50 new lines).